### PR TITLE
Generic method to get attribute values from LDAP

### PR DESCRIPTION
**Pull request by abpai94**
#31

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" colors="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/11.0/phpunit.xsd">
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" colors="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/11.0/phpunit.xsd">  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (coudot) [inline phpunit.xml:2]: Please remove this change

File: src/Ltb/Ldap.php [hunk 2]
```diff
@@ -259,23 +259,33 @@ function sorted_search($ldap_base, $ldap_filter, $attributes, $sortby, $ldap_siz
...
      */
-    function get_password_value($dn, $pwdattribute) {
-        $search_userpassword = \Ltb\PhpLDAP::ldap_read($this->ldap, $dn, "(objectClass=*)", array($pwdattribute));
-        if ($search_userpassword) {
-            $password_values = \Ltb\PhpLDAP::ldap_get_values($this->ldap, \Ltb\PhpLDAP::ldap_first_entry($this->ldap, $search_userpassword), $pwdattribute);
-            if(isset($password_values[0]))
-            {
-                return $password_values[0];
-            }
+    function get_attribute_value($dn, $attribute) {
```
Comment (coudot) [inline src/Ltb/Ldap.php:267]: It would be better to call it `get_attribute_values`

**Review by coudot — CHANGES_REQUESTED** [review]

File: phpunit.xml (hunk 1 above)
Comment (coudot) [reply phpunit.xml:2]:
```suggestion
<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" colors="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/11.0/phpunit.xsd">
```

**Review by coudot — APPROVED** [review]

**Comment by coudot** [comment]
We should also add a unit test in tests/Ltb/LdapTest.php

**Comment by davidcoutadeur** [comment]
I'll give a look to this feature on tomorrow.

@abpai94 : if the feature is finished, could your 1/ squash your commits and 2/ rebase on top of master? Thanks

**Review by davidcoutadeur — CHANGES_REQUESTED** [review]
Ok for me.

Last thing: we need to add the unit test for the new get_attribute_values function.

You can give a look to function test_get_password_value in file tests/Ltb/LdapTest.php

**Comment by abpai94** [comment]
@davidcoutadeur tests added, ready to merge!

**Review by davidcoutadeur — APPROVED** [review]

**Pull request merged**